### PR TITLE
Add tests for services.

### PR DESCRIPTION
**Pull request by svitlana-tertyshna**

### CODE REVIEW

File: src/test/java/core/basesyntax/model/FruitTransactionTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import org.junit.Test;
+
+public class FruitTransactionTest {
+    @Test
+    public void getByCode_ok() {
+        String code = "b";
+        FruitTransaction.Operation expected = FruitTransaction.Operation.BALANCE;
+        FruitTransaction.Operation actual = FruitTransaction.Operation.getByCode(code);
+        assertEquals(actual, expected);
```
Comment (nacenik) [inline src/test/java/core/basesyntax/model/FruitTransactionTest.java:13]:
```suggestion
        assertEquals(expected, actual);
```

File: src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String TITLE = "fruit,quantity";
+    private static final String SEPARATOR = ",";
+    private static final Map<String, Integer> STORAGE = FruitStorage.fruits;
```
Comment (nacenik) [inline src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java:11]: you can use FruitStorage.fruits directly. You don't need to create additional variable

File: src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.service.ReportService;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
```
Comment (nacenik) [inline src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java:11]: add method to test report if storage is empty

File: src/test/java/core/basesyntax/service/impl/TransactionParserImplTest.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionParser;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TransactionParserImplTest {
```
Comment (nacenik) [inline src/test/java/core/basesyntax/service/impl/TransactionParserImplTest.java:12]: add test if you data from file is empty and null

**Review by nacenik — CHANGES_REQUESTED** [review]

**Review by Ivan95kos — APPROVED** [review]
Great job
